CHANGELOG.next.asciidoc

```diff
@@ -71,6 +71,8 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 - Check for nil metadata in GCP. {pull}32281[32281]
 - update kubernetes controllermanager metricset to not collect deprecated metrics and fix dashboard {pull}32037[32037]
 - Fix to ARN parsing for Cloudwatch resource names with leading slashes {pull}32358[32358]
+- update elasticsearch node_stats metricset to use keyword for cgroup memory instead of long {pull}32197[32197]
```

Comment:
```suggestion
```

This looks like it was added as part of merge conflict resolution and can be removed.